Gemfile

```diff
@@ -4,7 +4,7 @@ gem 'cocoapods'
 
 # A reference to Maze Runner is only needed for running tests locally and if committed it must be
 # portable for CI, e.g. a specific release.  However, leaving it commented out would mean quicker CI.
-gem 'bugsnag-maze-runner', '~> 7.0'
+gem 'bugsnag-maze-runner', '~> 7.30.2'
```

Comment:
Why this change?  It means that you won't automatically get 7.31.0 and greater in future, but maybe that's what you want.